README.md

```diff
@@ -1,299 +1,93 @@
 [![][kong-logo]][kong-url]
 
-[![Build Status][badge-action-image]][badge-action-url]
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Kong/kong/blob/master/LICENSE)
-[![Twitter](https://img.shields.io/twitter/follow/thekonginc.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=thekonginc)
-
-Kong is a cloud-native, fast, scalable, and distributed Microservice
-Abstraction Layer *(also known as an API Gateway or API Middleware)*.
-Made available as an open-source project in 2015, its core values are
-high performance and extensibility.
-
-Actively maintained, Kong is widely used in production at companies ranging
-from startups to Global 5000 as well as government organizations.
-
-[Installation](https://konghq.com/install/#kong-community) |
-[Documentation](https://docs.konghq.com) |
-[Forum](https://discuss.konghq.com) |
-[Blog](https://konghq.com/blog) |
-IRC (freenode): [#kong](https://webchat.freenode.net/?channels=kong) |
-[Master Builds][kong-master-builds]
-
-## Summary
-
-- [**Why Kong?**](#why-kong)
-- [**Features**](#features)
-- [**Distributions**](#distributions)
-- [**Development**](#development)
-- [**Enterprise Support & Demo**](#enterprise-support--demo)
-- [**License**](#license)
-
-## Why Kong?
-
-If you are building for the web, mobile, or IoT (Internet of Things) you will
-likely end up needing common functionality to run your actual software. Kong
-can help by acting as a gateway (or a sidecar) for microservices requests while
-providing load balancing, logging, authentication, rate-limiting,
-transformations, and more through plugins.
+![Stars](https://img.shields.io/github/stars/Kong/kong?style=flat-square) ![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Kong/kong?style=flat-square) ![Docker Pulls](https://img.shields.io/docker/pulls/_/kong?style=flat-square) [![Build Status][badge-action-image]][badge-action-url] ![Version](https://img.shields.io/github/v/release/Kong/kong?color=green&label=Version&style=flat-square)  ![License](https://img.shields.io/badge/License-Apache%202.0-blue?style=flat-square)
```

Comment:
Very minor, but looking at these badges again, I am thinking the order could be improved. I'd say the main ones are
Build Status, Version, and License; perhaps they could come first?